<launch>
    <param name="name" value="yoon1"/>
    <node pkg="tcps_image_attack" type="node_car_airsim.py" name="car_airsim_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_yolo.py" name="yolo_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_car_controller.py" name="car_controller_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_image_attack_trainer.py" name="image_attack_train_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_image_attacker.py" name="image_attack_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_car_highlevel_decision_maker.py" name="high_level_decision_maker" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_highlevel_decision_trainer.py" name="high_level_decision_trainer" output="screen">
    </node>
</launch>
